.github/workflows/ci.yaml

```diff
@@ -0,0 +1,29 @@
+name: ci
+
```

Comment:
Do we need to specify which branch this action is taken on?

Reply:
This will run on any branch, which I think is what we want. When I create a new branch for a new issue, I would want the build to run on it.